### DISCUSSION TITLE
Windows 11 Edge path check

### DISCUSSION THREAD

**Discussion by regorxxx** [TT-ReBORN/Georgia-ReBORN#84] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/83307074/198391301-a511fc3c-65b8-4580-80af-489be07c3ca3.png)
@Wil-B 
@TT-ReBORN 

Just in case someone can help with this.

**Comment by TT-ReBORN** [comment]
@Zephyr0ck and @takaji10 or someone else,

Can you guys check on your Win11 if there is Internet Explorer installed in
C:\Program Files\
C:\Program Files (x86)\

and if Edge is installed here with correct path:
C:\Program Files\ ???
C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe

Thanks!

-Tom

**Reply by Zephyr0ck** [reply, marked as answer]
> Edition	Windows 11 Pro
Version	21H2
OS build	22000.1098
Experience	Windows Feature Experience Pack 1000.22000.1098.0

`C:\Program Files\Internet Explorer`
`C:\Program Files (x86)\Internet Explorer`

<blockquote class="imgur-embed-pub" lang="en" data-id="giMH89V"><a href="https://imgur.com/giMH89V">View post on imgur.com</a></blockquote>
<blockquote class="imgur-embed-pub" lang="en" data-id="bz8gsF0"><a href="https://imgur.com/bz8gsF0">View post on imgur.com</a></blockquote>

Yep, it's available in Windows 11 yet

**Reply by TT-ReBORN** [reply]
@Zephyr0ck, 

José, what about Edge?

Do you have x64 version in 
`C:\Program Files\ ???`

and x86 version in:
`C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe`

I don't know about these paths, don't have Win11 only this info from internet... please confirm, thanks!

-Tom

**Reply by Zephyr0ck** [reply]
it's located into:

`C:\Windows\SystemApps\Microsoft.MicrosoftEdge_8wekyb3d8bbwe`
`C:\Program Files (x86)\Microsoft\Edge\Application`

I'm running Windows 11 x64 and there's no folder for Edge into **C:\Program Files**
